### PR TITLE
[5/?] Add ability to run code examples in the playground: Add CORS exception for tutorial.ponylang.io to route /evaluate.json

### PR DESCRIPTION
**Pull request by shaedrich**
See https://github.com/ponylang/pony-tutorial/issues/340

As discussed in #205

### CODE REVIEW

**Comment by jemc** [comment]
Looks good to me. I can approve after it's no longer a draft.

**Review by jemc — APPROVED** [review]

File: src/api.rs [hunk 1]
```diff
@@ -7,6 +7,12 @@ use axum::{
 use crate::routes::{compile, create_gist, evaluate, static_css, static_html, static_js};
 use crate::GithubClient;
 use std::net::SocketAddr;
+use http::HeaderValue;
+use tower_http::cors::CorsLayer;
+
+let layer = CorsLayer::new().allow_origin(
```
Comment (jemc) [inline src/api.rs:13]: Looks like the linter wants this change:
```suggestion
const layer = CorsLayer::new().allow_origin(
```

**Comment by jemc** [comment]
Looks like there's a parsing issue in the CI failure.

**Comment by shaedrich** [comment]
It's fixed now.

**Comment by shaedrich** [comment]
Done

**Comment by SeanTAllen** [comment]
@mfelsche do you know what needs to happen for this?

**Review by mfelsche — CHANGES_REQUESTED** [review]
This still needs a dependency on `tower_http` added to `Cargo.toml` like so:

```toml
tower-http = { version = "0.5", features = ["cors"] }
```

File: src/api.rs [hunk 2]
```diff
@@ -6,7 +6,15 @@ use axum::{
 
 use crate::routes::{compile, create_gist, evaluate, static_css, static_html, static_js};
 use crate::GithubClient;
+use http::HeaderValue;
 use std::net::SocketAddr;
+use tower_http::cors::CorsLayer;
+
+const layer: CorsLayer = CorsLayer::new().allow_origin(
```
[the remark this answers is not in the document]
Comment (shaedrich) [reply src/api.rs]: Done

**Review by mfelsche — APPROVED** [review]
LGTM

**Comment by mfelsche** [comment]
@shaedrich this is live now on https://playground.ponylang.io

**Comment by shaedrich** [comment]
Cool, thanks for deploying!

**Pull request merged**